src/SkprConfig.php

```diff
@@ -79,14 +79,20 @@ public function load(string $filename = self::CONFIG_FILENAME): self {
         return $this;
       }
       $this->config = json_decode($data, TRUE);
-      array_walk($this->config, function ($value, $key) {
-        // Store as env vars.
-        putenv($this->convertToEnvVarName($key) . '=' . $value);
-      });
     }
     return $this;
   }
 
+  /**
+   * Puts all Skpr config into env vars.
+   */
+  public function putEnvs() {
```

Comment:
Can we extend this with an argument so that we can export the specific config keys we're interested in? Maybe default to NULL for everything?

Reply:
Why not just call `putenv()` yourself for the ones you are interested in?

Reply:
Definitely an option :)

Reply:
OK. Added this feature. Let me know what you think.